homeassistant/components/netatmo.py

```diff
@@ -50,7 +49,4 @@ def setup(hass, config):
         _LOGGER.error("Unable to connect to NatAtmo API")
         return False
 
-    for component in 'camera', 'sensor':
```

Comment:
Other platforms fix this by checking the data to see if there is a camera, sensor etc.

I want to keep the discovery in because it is helps users get started. If the camera component will only setup 1 camera, then that is also that has to be fixed.